### PR TITLE
Add direct link to where quiz data lives

### PR DESCRIPTION
**Pull request by erictleung**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Thought it would be easier for contributors to find where the data lives by putting it directly in the README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ This open source project is a work in progress and ever evolving.
 
 We welcome all contributions, suggestions and ideas for improvement from the community.
 
-You can contribute by adding  new quiz questions or making changes to the code.
+You can contribute by adding new quiz questions or making changes to the code [here](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/full-quiz.js).
```
Comment (jdwilkin4) [inline README.md:47]: Proposed change:

You can contribute by fixing bugs in the codebase, proposing new features or adding new questions to the [quiz file](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/full-quiz.js).

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @erictleung ! 
Thank you for your PR.

Left a comment for a proposed change to the readme 👍

File: README.md (hunk 1 above)
Comment (erictleung) [reply README.md:47]:
```suggestion
You can contribute by fixing bugs in the codebase, proposing new features or adding new questions to the [quiz file](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/full-quiz.js).
```

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Pull request merged**